index.bs

```diff
@@ -1050,6 +1052,7 @@ Possible values are:
 <ul dfn-for="source debug data type">
 <li>"<dfn><code>source-destination-limit</code></dfn>"
 <li>"<dfn><code>source-destination-rate-limit</code></dfn>"
+<li>"<dfn><code>source-output-space-limit</code></dfn>"
```

Comment:
I'm not sure if it means we should change this but a high number of states can be associated with info gain within the limit.